easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml

```
@@ -29,3 +29,6 @@ easyconfigs:
       # see https://github.com/easybuilders/easybuild-easyconfigs/pull/19554
       options:
         from-pr: 19554
+  - dask-2023.9.2-foss-2023a.eb:
+      options:
+        from-pr: 19996
```

I'm not 100% pleased with this as this PR actually has nothing to do with `dask`. It would have been better to explicitly call out the easyconfig we were using from the PR.
```suggestion
  - Pillow-SIMD-9.5.0-GCCcore-12.3.0.eb:
      options:
        from-pr: 19996
  - dask-2023.9.2-foss-2023a.eb
```